python_example/consumer.py

```diff
@@ -0,0 +1,48 @@
+import time
+import os
+from confluent_kafka import Consumer
+
+import requests
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def _get_token(config):
+    payload = {
+        'grant_type': 'client_credentials',
+        'scope': 'api.iam.service_accounts'
+    }
+    resp = requests.post(os.getenv("RHOAS_SERVICE_ACCOUNT_OAUTH_TOKEN_URL"),
```

Comment:
can we move all the `os.getenv` at the top so that we create a natural flow:

- collecting all the input
- sign in
- opening connection
- reading